Woodog/Woodog/settings.py

```diff
@@ -43,6 +43,7 @@
     'woodogdata',
     'authentication',
     'appdashboard',
+    'froala_editor'
```

Comment:
why this?

Reply:
Using froala editor in django in place of the regular django Textarea editor we can get user input with, files and even videos

Reply:
Okay